src/main/java/com/gmail/axelwerst/Shape.java

```diff
@@ -0,0 +1,76 @@
+package com.gmail.axelwerst;
+
+    abstract class Shape {
+        public abstract double calculateArea();
+        public abstract double calculatePerimeter();
+
+        public abstract String displayInfo();
+    }
+    class Circle extends Shape {
```

Comment:
Classes `Circle`. `Rectangle` and `Sqaure` must be in separate files as we try to follow the convention one class in one file.